<html><h1>Validation FAILED.</h1><p>Build Time: 22 minutes 20 seconds 344 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/fa8174a94a593653a535cb1a38984d907f449938">fa8174a94a593653a535cb1a38984d907f449938</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 1 formatting issues:
     [echo] 1: ./commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceChannelLocalServiceImpl.java expected:&lt;...xception.class});
     [echo] 
     [echo] }[]&gt; but was:&lt;...xception.class});
     [echo] 
     [echo] }[
     [echo] ]&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [echo] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [echo] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/90//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/90//testReport">here</a>.</h5></div></html>

--- a/commerce-frontend-api/bnd.bnd
+++ b/commerce-frontend-api/bnd.bnd
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Commerce Frontend API
 Bundle-SymbolicName: com.liferay.commerce.frontend.api
-Bundle-Version: 1.3.0
+Bundle-Version: 2.0.0
 Export-Package:\
 	com.liferay.commerce.frontend,\
 	com.liferay.commerce.frontend.model,\

--- a/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableContextContributorRegistry.java
+++ b/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableContextContributorRegistry.java
@@ -24,7 +24,7 @@ import java.util.List;
 @ProviderType
 public interface ClayTableContextContributorRegistry {
 
-	public List<ClayTableContextContributor> getClayTableContextContributors(
-		String key);
+	public List<ClayTableHttpContextContributor>
+		getClayTableContextContributors(String key);
 
 }

--- a/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableHttpContextContributor.java
+++ b/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableHttpContextContributor.java
@@ -25,7 +25,7 @@ import javax.servlet.http.HttpServletRequest;
  * @author Marco Leo
  */
 @ProviderType
-public interface ClayTableContextContributor {
+public interface ClayTableHttpContextContributor {
 
 	public void contribute(
 		ClayTable clayTable, Map<String, Object> context,

--- a/commerce-frontend-api/src/main/resources/com/liferay/commerce/frontend/packageinfo
+++ b/commerce-frontend-api/src/main/resources/com/liferay/commerce/frontend/packageinfo
@@ -1,1 +1,1 @@
-version 1.1.0
+version 2.0.0

--- a/commerce-frontend-impl/build.gradle
+++ b/commerce-frontend-impl/build.gradle
@@ -7,9 +7,7 @@ dependencies {
 
 	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.account.api", version: "3.6.0"
-	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.api", version: "9.3.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.currency.api", version: "2.0.0"
-	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.frontend.api", version: "1.2.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.wish.list.api", version: "3.0.0"
 	compileOnly group: "com.liferay.portal", name: "release.portal.api", version: "7.1.1"
 	compileOnly group: "javax.portlet", name: "portlet-api", version: "3.0.0"
@@ -21,5 +19,7 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.jaxrs", version: "1.0.0"
+	compileOnly project(":apps:commerce:commerce-api")
+	compileOnly project(":apps:commerce:commerce-frontend-api")
 	compileOnly project(":apps:commerce:commerce-product-api")
 }

--- a/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/clay/table/ClayTableContextContributorRegistryImpl.java
+++ b/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/clay/table/ClayTableContextContributorRegistryImpl.java
@@ -14,8 +14,8 @@
 
 package com.liferay.commerce.frontend.internal.clay.table;
 
-import com.liferay.commerce.frontend.ClayTableContextContributor;
 import com.liferay.commerce.frontend.ClayTableContextContributorRegistry;
+import com.liferay.commerce.frontend.ClayTableHttpContextContributor;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerCustomizerFactory;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerCustomizerFactory.ServiceWrapper;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
@@ -43,10 +43,10 @@ public class ClayTableContextContributorRegistryImpl
 	implements ClayTableContextContributorRegistry {
 
 	@Override
-	public List<ClayTableContextContributor> getClayTableContextContributors(
-		String key) {
+	public List<ClayTableHttpContextContributor>
+		getClayTableContextContributors(String key) {
 
-		List<ServiceWrapper<ClayTableContextContributor>>
+		List<ServiceWrapper<ClayTableHttpContextContributor>>
 			clayTableContextContributorServiceWrappers =
 				ListUtil.fromCollection(_serviceTrackerMap.getService(key));
 
@@ -60,27 +60,27 @@ public class ClayTableContextContributorRegistryImpl
 			return Collections.emptyList();
 		}
 
-		List<ClayTableContextContributor> clayTableContextContributors =
+		List<ClayTableHttpContextContributor> clayTableHttpContextContributors =
 			new ArrayList<>();
 
-		for (ServiceWrapper<ClayTableContextContributor>
+		for (ServiceWrapper<ClayTableHttpContextContributor>
 				tableActionProviderServiceWrapper :
 					clayTableContextContributorServiceWrappers) {
 
-			clayTableContextContributors.add(
+			clayTableHttpContextContributors.add(
 				tableActionProviderServiceWrapper.getService());
 		}
 
-		return clayTableContextContributors;
+		return clayTableHttpContextContributors;
 	}
 
 	@Activate
 	protected void activate(BundleContext bundleContext) {
 		_serviceTrackerMap = ServiceTrackerMapFactory.openMultiValueMap(
-			bundleContext, ClayTableContextContributor.class,
+			bundleContext, ClayTableHttpContextContributor.class,
 			"commerce.table.name",
 			ServiceTrackerCustomizerFactory.
-				<ClayTableContextContributor>serviceWrapper(bundleContext));
+				<ClayTableHttpContextContributor>serviceWrapper(bundleContext));
 	}
 
 	@Deactivate
@@ -92,7 +92,7 @@ public class ClayTableContextContributorRegistryImpl
 		ClayTableContextContributorRegistryImpl.class);
 
 	private ServiceTrackerMap
-		<String, List<ServiceWrapper<ClayTableContextContributor>>>
+		<String, List<ServiceWrapper<ClayTableHttpContextContributor>>>
 			_serviceTrackerMap;
 
 }

--- a/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/clay/table/comparator/ClayTableContextContributorOrderComparator.java
+++ b/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/clay/table/comparator/ClayTableContextContributorOrderComparator.java
@@ -14,7 +14,7 @@
 
 package com.liferay.commerce.frontend.internal.clay.table.comparator;
 
-import com.liferay.commerce.frontend.ClayTableContextContributor;
+import com.liferay.commerce.frontend.ClayTableHttpContextContributor;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerCustomizerFactory.ServiceWrapper;
 import com.liferay.portal.kernel.util.MapUtil;
 
@@ -26,7 +26,7 @@ import java.util.Comparator;
  * @author Marco Leo
  */
 public class ClayTableContextContributorOrderComparator
-	implements Comparator<ServiceWrapper<ClayTableContextContributor>>,
+	implements Comparator<ServiceWrapper<ClayTableHttpContextContributor>>,
 			   Serializable {
 
 	public ClayTableContextContributorOrderComparator() {
@@ -39,8 +39,8 @@ public class ClayTableContextContributorOrderComparator
 
 	@Override
 	public int compare(
-		ServiceWrapper<ClayTableContextContributor> serviceWrapper1,
-		ServiceWrapper<ClayTableContextContributor> serviceWrapper2) {
+		ServiceWrapper<ClayTableHttpContextContributor> serviceWrapper1,
+		ServiceWrapper<ClayTableHttpContextContributor> serviceWrapper2) {
 
 		int displayOrder1 = MapUtil.getInteger(
 			serviceWrapper1.getProperties(),

--- a/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CommerceTableTag.java
+++ b/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CommerceTableTag.java
@@ -15,9 +15,9 @@
 package com.liferay.commerce.frontend.taglib.servlet.taglib;
 
 import com.liferay.commerce.frontend.ClayTable;
-import com.liferay.commerce.frontend.ClayTableContextContributor;
 import com.liferay.commerce.frontend.ClayTableContextContributorRegistry;
 import com.liferay.commerce.frontend.ClayTableDataJSONBuilder;
+import com.liferay.commerce.frontend.ClayTableHttpContextContributor;
 import com.liferay.commerce.frontend.ClayTableRegistry;
 import com.liferay.commerce.frontend.ClayTableSerializer;
 import com.liferay.commerce.frontend.CommerceDataProviderRegistry;
@@ -308,11 +308,11 @@ public class CommerceTableTag extends ComponentRendererTag {
 
 		Set<String> dependencies = new HashSet<>();
 
-		List<ClayTableContextContributor> clayTablePostProcessors =
+		List<ClayTableHttpContextContributor> clayTablePostProcessors =
 			_clayTableContextContributorRegistry.
 				getClayTableContextContributors(tableName);
 
-		for (ClayTableContextContributor clayTableContextContributor :
+		for (ClayTableHttpContextContributor clayTableContextContributor :
 				clayTablePostProcessors) {
 
 			clayTableContextContributor.contribute(

--- a/commerce-theme-minium/commerce-theme-minium-impl/build.gradle
+++ b/commerce-theme-minium/commerce-theme-minium-impl/build.gradle
@@ -11,7 +11,6 @@ dependencies {
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.api", version: "9.3.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.cart.taglib", version: "2.0.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.currency.api", version: "2.0.0"
-	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.frontend.api", version: "1.2.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.frontend.taglib", version: "2.1.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.inventory.api", version: "1.2.0"
 	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.price.list.api", version: "4.0.0"
@@ -29,6 +28,7 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.cm", version: "1.5.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly project(":apps:commerce:commerce-frontend-api")
 	compileOnly project(":apps:commerce:commerce-product-api")
 
 	jsCompile group: "com.liferay", name: "com.liferay.frontend.taglib.soy", version: "2.0.0"

--- a/commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java
+++ b/commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java
@@ -15,7 +15,7 @@
 package com.liferay.commerce.theme.minium.internal.table.contributor;
 
 import com.liferay.commerce.frontend.ClayTable;
-import com.liferay.commerce.frontend.ClayTableContextContributor;
+import com.liferay.commerce.frontend.ClayTableHttpContextContributor;
 import com.liferay.frontend.js.loader.modules.extender.npm.NPMResolver;
 import com.liferay.portal.kernel.theme.ThemeDisplay;
 import com.liferay.portal.kernel.util.WebKeys;
@@ -40,10 +40,10 @@ import org.osgi.service.component.annotations.Reference;
 		"commerce.table.name=commerceAccounts",
 		"commerce.table.name=commerceAccountUsers"
 	},
-	service = ClayTableContextContributor.class
+	service = ClayTableHttpContextContributor.class
 )
 public class CommerceAccountClayTableContextContributor
-	implements ClayTableContextContributor {
+	implements ClayTableHttpContextContributor {
 
 	@Override
 	public void contribute(

--- a/source-formatter-suppressions.xml
+++ b/source-formatter-suppressions.xml
@@ -24,7 +24,6 @@
 		<suppress checks="JavaAPISignatureCheck" files="commerce-api/src/main/java/com/liferay/commerce/starter/CommerceRegionsStarter\.java" />
 		<suppress checks="JavaAPISignatureCheck" files="commerce-api/src/main/java/com/liferay/commerce/util/CommerceCheckoutStep\.java" />
 		<suppress checks="JavaAPISignatureCheck" files="commerce-api/src/main/java/com/liferay/commerce/util/CommerceCheckoutStepServicesTracker\.java" />
-		<suppress checks="JavaAPISignatureCheck" files="commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableContextContributor\.java" />
 		<suppress checks="JavaAPISignatureCheck" files="commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableDataJSONBuilder\.java" />
 		<suppress checks="JavaAPISignatureCheck" files="commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/CommerceDataSetDataProvider\.java" />
 		<suppress checks="JavaAPISignatureCheck" files="commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/FilterFactory\.java" />

